1750 start date and throw scmdata back (#21)
openscm/openscm-runner: CHANGELOG.rst

master
------

> - (`#21 <https://github.com/openscm/openscm-runner/pull/21>`_) Flexible start date for FaIR; build scmdata to emissions converter on openscm-runner

Reviewer: ```suggestion
- (`#21 <https://github.com/openscm/openscm-runner/pull/21>`_) Added flexible start date for FaIR and FaIR's scmdata to emissions converter
```
